example/credentials.yml

```
@@ -127,43 +131,31 @@ property:
   - id: serviceEndpoint
     label: service endpoint
     domain: cred:RefreshService
+    range: IRI
     comment: The value of the `serviceEndpoint` property MUST be a URL to the service endpoint associated with the subject.
 
   - id: termsOfUse
     label: terms of use
-    domain: cred:VerifiableCredential
+    domain: cred:VerifiableCredential,cred:VerifiablePresentation
     range: odrl:Policy
     comment: |
-      If specified, the value of the `termsOfUse` property MUST specify one or more terms of use policies under which the creator issued the credential or presentation. If the recipient (a holder or verifier) is not willing to adhere to the specified terms of use, then they do so on their own responsibility and might incur legal liability if they violate the stated Terms of Use. Each `termsOfUse` MUST specify its type, for example, `IssuerPolicy`, and optionally, its instance `id`. The precise contents of each term of use is determined by the specific `TermsOfUse` type definition.
+      If specified, the value of the optional `termsOfUse` property MUST specify one or more terms of use policies under which the issuer issued the credential or presentation. Each `termsOfUse` policy MUST specify its type (for example, `IssuerPolicy`) and MAY specify its instance `id`. The precise content of each term of use is determined by the specific `TermsOfUse` type definition. If the recipient (a holder or verifier) violates the specified terms of use, the responsibility is their own, and such violation may incur legal liability.
+
+  - id: validFrom
+    label: Valid from
+    domain: cred:VerifiableCredential
+    range: xsd:dateTime
+    comment: |
+      The value of the `validFrom` property MUST be a string value of an ISO8601 combined date and time string representing the date and time the credential was issued. Note that this date represents the earliest date when the information associated with the <a href="#credentialSubject>`credentialSubject`</a> property became valid.
```

I thought we concluded that `validFrom` was disconnected from the issuance date-time, and that it could be in the future?
```suggestion
      The value of the `validFrom` property MUST be a string value of an ISO8601 combined date and time string representing the earliest date and time the information associated with the <a href="#credentialSubject>`credentialSubject`</a> property became (or will become) valid.
```

---

@TallTed, whilst these are valid comments, it is not for this PR nor indeed repository. This yml file is a local _copy_ of the "official" yml definition to test, and preview the results of, the changes in the software.

If and when this PR is accepted and merged, the "real" yml file will have to be updated as well. Ie, I will issue a new PR _on that repository_ with the changes that are required by these changes, although those changes will be on the HTML formatting only. You may want to make these comments on that PR or (and maybe preferably) as a separate set of change requests on the security vocabulary which, frankly, does require a lot of editorial (and non-editorial) updates which are pending.

---

> @iherman — [My review was requested on this PR](https://github.com/w3c/yml2vocab/pull/9#event-8718905084)... so I provided it.

And I thank you for this...

> 
> It is not obvious to me when artifacts of CI or similar are managed by git, but other files should be edited to be reflected in those artifacts.
> 
> I would _LOVE_ to automagically know which files I should ignore (because it would save me that time and effort) ... but shouldn't those just be listed in `.gitignore`, and so not show up as "Changed files" when I try to review the files that matter?

In this very case I should have made this explicit manually; sorry about that. I will modify the issue description.

The reason I added the vocabulary examples is that you, or any reviewer, could look at the result of the code changes, ie, how the final vocabulary _would_ look like in case this code change is accepted. Had I put them into `.gitignore`, the only thing you would have looked at is the TypeScript files that have changed (and the README).